libraries/helpers.rb

```diff
@@ -0,0 +1,22 @@
+module RabbitMQCookbook
+  module Helpers
+    require 'mixlib/shellout'
+
+    def remove_self_from_cluster
```

Comment:
I think that if anything we should make this more verbose than even my initial naming. Here are some suggestions:
- `remove_self_from_rmq_cluster`
- `reset_rmq_self`
- `leave_rmq_cluster`

Let me know which you would prefer or suggest an alternative that is clear as to its purpose both internally within this cookbook and be called in the context of another cookbook.